### ISSUE TITLE
repo->user: Extensions do not get automatically installed from remote settings

### ISSUE THREAD

**Issue by seppzer0** [zokugun/vscode-sync-settings#55]
Hi!

First of all -- I want to say that I appreciate this project for providing VSCode's Sync Settings alternative for VSCodium. There are not many in existence.

I'm currently encountering the issue of my extensions not get automatically installed when doing the "repository -> user" sync.
The settings in my repository contain all of the information (including extentions' information) from my primary setup on my Linux machine.
However, when trying to pull these settings on my Windows machine, none of the mentioned extensions are installed.

I have looked briefly through the existing issues in this project, but didn't find anything that can be related to this.
I can only assume that I am missing something, which is why your help would be much appreciated!

**Comment by daiyam** [comment]
Do you have any errors in `View`/`Output`/`Sync Settings` or `View`/`Output`/`Extension Host`?

**Comment by seppzer0** [comment]
> Do you have any errors in `View`/`Output`/`Sync Settings` or `View`/`Output`/`Extension Host`?

No errors. Only `info` messages.
I did however find the underlying reason of my issue. And it was a simple user error on my end.

The profile name that I saved in my repository had a custom name (let's call it `primary`). The profile name I had in local settings on my Windows machine was at it's default value, which is `main`. This conflict resulted in a confusing behaviour, which is that each time when the "repo->user" sync was launched, the Sync Settings extension brought my VSCodium installation to a blank state.

Maybe it would make sense to add an extra check on the presence of the profile name specified in Sync Settings' `settings.yml`, when doing the "repo->user" sync? Of course, it is ultimately a user's fault of not checking the correct value, but having an extra user warning is always nice in my opinion.

In any case, I'm closing this issue.
Even if it was my own usage error, I appreciate your time looking into this!
